extensions/vscode/src/extension.ts

```diff
@@ -43,11 +49,24 @@ export async function activate(context: ExtensionContext) {
   setStateContext(PositPublishState.uninitialized);
   setInitializationInProgressContext(InitializationInProgress.false);
 
-  const port = await ports.acquire();
+  const useExternalAgent =
+    context.extensionMode === ExtensionMode.Development &&
+    process.env.POSIT_PUBLISHER_USE_EXTERNAL_AGENT === "TRUE";
+  console.log(
+    "Starting Context in extension mode: %s, with useExternalAgent set to %s",
+    context.extensionMode,
+    useExternalAgent,
+  );
+
+  let port = await ports.acquire();
```

Comment:
Should we skip acquiring a port if we are using an external agent?

Reply:
I can swap the logic yes. That would maintain that port available for other consumers, so it is a good point.